tests/conftest.py

```
@@ -1,190 +1,346 @@
 import pytest
+import pytest_asyncio
 import asyncio
-import tempfile
-import os
+from typing import AsyncGenerator, Generator
```

Ruff (0.11.9)

`typing.Generator` imported but unused

Remove unused import: `typing.Generator`

(F401)